src/collective/cover/browser/cover.py

```diff
@@ -279,15 +279,15 @@ def __call__(self):
 
 class UpdateTile(grok.View):
     grok.context(ICover)
-    grok.require('cmf.ModifyPortalContent')
+    grok.require('zope2.View')
 
     def render(self):
         tile_type = self.request.form.get('tile-type')
         tile_id = self.request.form.get('tile-id')
-
-        if tile_type and tile_id:
-            tile = self.context.restrictedTraverse(tile_type)
-            tile_instance = tile[tile_id]
+        if not all([tile_type, tile_id]):
```

Comment:
you don't get my point: I can call this view with invalid values of `tile-type` and/or `tile-id` and it will rise exceptions other than `BadRequest`: `AttributeError`, just to mention one.